Kickstarter-iOS/Features/RewardsCollection/Controller/WithShippingRewardsCollectionViewController.swift

```diff
@@ -193,11 +193,10 @@ final class WithShippingRewardsCollectionViewController: UICollectionViewControl
 
     self.viewModel.outputs.rewardsCollectionViewIsHidden
       .observeForUI()
-      .observeValues { [weak self] shimmerLoadingViewIsHidden in
+      .observeValues { [weak self] rewardsCollectionViewIsHidden in
         guard let self else { return }
 
-        self.collectionView.isHidden = !shimmerLoadingViewIsHidden
-        self.collectionView.layoutIfNeeded()
```

Comment:
I deleted this because now that we're not always waiting for the location view to load, it gets called before the view hierarchy is ready. I did test a bunch of random projects (hence the local pickup bug I found) and didn't see any UI weirdness caused by removing this line.